### PR TITLE
feat(pci-kubernetes): add cluster infos tile

### PR DESCRIPTION
**Pull request by seven-amid**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/pci-kubernetes
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | DTCORE-2450
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [ ] Commits are signed-off
- [ ] Only FR translations have been updated
- [ ] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/pci-kubernetes/src/components/information/ClusterInformation.component.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    const {
+      privateNetworkRoutingAsDefault,
+      defaultVrackGateway,
+    } = kubeDetail?.privateNetworkConfiguration;
+
+    return !privateNetworkRoutingAsDefault && defaultVrackGateway;
+  };
+
+  if (isPending) {
+    return null;
```
Comment (kqesar) [inline packages/manager/apps/pci-kubernetes/src/components/information/ClusterInformation.component.tsx:53]: Maybe add a spinner instead of null ?

**Review by kqesar — APPROVED** [review]

File: packages/manager/apps/pci-kubernetes/src/api/hooks/useKubernetes.ts [hunk 2]
```diff
@@ -81,3 +81,46 @@ export const useKubernetesCluster = (projectId: string, kubeId: string) =>
     queryKey: getKubernetesClusterQuery(projectId, kubeId),
     queryFn: () => getKubernetesCluster(projectId, kubeId),
   });
+
+export const useKube = (projectId: string, kubeId: string) => {
```
Comment (frenautvh) [inline packages/manager/apps/pci-kubernetes/src/api/hooks/useKubernetes.ts:85]: nitpick :

```suggestion
export const useKubeDetails = (projectId: string, kubeId: string) => {
```

File: packages/manager/apps/pci-kubernetes/src/components/information/ClusterInformation.component.tsx (hunk 1 above)
Comment (frenautvh) [inline packages/manager/apps/pci-kubernetes/src/components/information/ClusterInformation.component.tsx:53]: return spinner ?

Comment (seven-amid) [reply packages/manager/apps/pci-kubernetes/src/components/information/ClusterInformation.component.tsx:34]: In production actually, the page including this Tile is not rendering while the detail fetching is pending, so no need for spinner/skeletons in this case

**Review by frenautvh — APPROVED** [review]

**Pull request merged**